app/qml/ActiveLayerPanel.qml

```diff
@@ -22,6 +42,22 @@ Drawer {
         color: InputStyle.clrPanelMain
     }
 
+    Item {
+        id: stateManager
+        states: [
+            State {
+                name: "setup"
+                PropertyChanges { target: layerPanel; title: "Default survey layer"}
+            }
+
+           ,State {
+                name: "record"
+                PropertyChanges { target: layerPanel; title: "Survey layer" }
```

Comment:
If we have just two states here and we only change title, maybe it would be easier to just use `title: state == "setup" ? "default survey layer" : "survey layer"` instead of using `PropertyChanges` but I don't have a strong opinion about that...